### PR TITLE
Added Filipino translate

### PR DESCRIPTION
**Pull request by maglionaire**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
    Feature to add i18n - Filipino Language Support
- **Why was this change needed?** (You can also link to an open issue here)
    Fixes issue 🚀 Feature: i18n - Filipino Language Support #1558 
- **Other information**:

### CODE REVIEW

File: packages/notification-center/src/i18n/languages/ph.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { ITranslationEntry } from '../lang';
+
+export const PH: ITranslationEntry = {
+  translations: {
+    notifications: 'Abiso',
+    markAllAsRead: 'Markahan ang lahat bilang',
```
Comment (AbbeyIT) [inline packages/notification-center/src/i18n/languages/ph.ts:6]: This should be "Markahan ang lahat ng bilang"

**Review by ShaneMaglangit — COMMENTED** [review]
Hey, I just have a couple of suggestions to improve this PR.

- [ ] FIL is the more appropriate/correct language code. PH refers to the country.
- [ ] Replace "Abiso" to "Notipikasyon"
- [ ] Replace "Markahan ang lahat bilang" to "Markahan ang lahat bilang nabasa"
- [ ] Replace "Pinatatakbo" with "Pinagagana"

## Justifications:

**Why "Notipikasyon"?**
It is important to note that Tagalog != Filipino. Abiso is the direct translation of Notification in Tagalog. Filipino is rather different as it a combination of Tagalog + inspiration from foreign languages (mainly English and Spanish). Abiso is too traditional, where it doesn't really align in this context of Notification as a technical feature. You can think of Abiso more aligned with notify/notified (an action).

Notipikasyon is an informal translation of Notification. It is more intuitive translation. 

**Why "Markahan ang lahat bilang nabasa"?**

This one is actually pretty simple, the current provided sentence actually only translates to "Mark all as". 

The suggested correction by @AbbeyIT translates to "Mark all counts". By adding "ng", the definition of `bilang` changes from "as" to "count".

Markahan => Mark
Ang Lahat => All
As => Bilang
Nabasa => Read

**Why "Pinagagana"?**

"Pinatatakbo ng" seems like the correct translation at first, even Google translate agrees. However, it somehow translates more towards "Operated by" in this specific context. Pinagagana better defines the context of the phrase.

To better defined the difference between two translation. Suppose there's a truck being driven by a guy.

(ang) driver (ay) pinatatakbo (ang) truck.
(ang) fuel (ay) pinapagana (ang) truck.

> ignore words enclosed in parenthesis, they are just linking/article words.

**Comment by AbbeyIT** [comment]
@ShaneMaglangit You should interchange ```As => Bilang``` into ```Bilang => As```. So they won't get confused.

**Comment by ShaneMaglangit** [comment]
> @ShaneMaglangit You should interchange `As => Bilang` into `Bilang => As`. So they won't get confused.

Corrected it, thanks for pointing it out!

**Comment by p-fernandez** [comment]
@ShaneMaglangit we are working with ISO 639-1 so only 2 character codes are allowed. I think they are not the same but maybe we should use tagalog iso code `tl`. If tagalog is not interchangeable with filipino, then we can't implement this language for now.

**Comment by ShaneMaglangit** [comment]
It depends on how we value correctness for the app.

In a strict formal sense, they are not interchangeable.

Colloquially however, we use Tagalog to refer to Filipino and "Pure Tagalog" for the traditional language. I don't see any problem using TL if we disregard technicalities.

On the other hand, there is actually no practical value in forcing support for Filipino/Tagalog.

English is almost our default language and we would have no problem navigating the web app with it.

If anything, it would be more challenging for us to use a web app in Filipino than in English.

**Comment by maglionaire** [comment]
> @p-fernandez It depends on how we value correctness for the app.
> 
> In a strict formal sense, they are not interchangeable.
> 
> Colloquially however, we use Tagalog to refer to Filipino and "Pure Tagalog" for the traditional language. I don't see any problem using TL if we disregard technicalities.
> 
> On the other hand, there is actually no practical value in forcing support for Filipino/Tagalog.
> 
> English is almost our default language and we would have no problem navigating the web app with it. We breathe and eat English.
> 
> If anything, it would be more challenging for us to use a web app in Filipino than in English.
I resolved it according to your suggestions.
Please correct me if I'm wrong and thank you for your suggestions.
But I don't know how to make you guys be the co-author.

**Pull request merged**